modules/_node-scripts/report/source/imports.mjs

```diff
@@ -0,0 +1,160 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import * as acorn from 'acorn';
+import tsPlugin from 'acorn-typescript';
+import estraverse from 'estraverse';
```

Comment:
These don't seem to be in our package.json?

Reply:
Yeah, I noticed it when running this PR locally again :roll_eyes:  

I have pushed new commits to fix that. thx.